### ISSUE TITLE
Check menu keyboard functionality. Tabs should close after tabbing moves off the menu item

### ISSUE THREAD

**Issue by robgasc** [web-illinois/illinois_framework_theme#1186]
**Describe the bug**
Check menu keyboard functionality. Tabs should close after tabbing moves off the menu item.

The Menu system provides some nice functionality. One item to mention is that the specific menu dropdowns do not collapse after tabbing beyond the dropdown. For instance, after tabbing through the “Research” menu, tabbing moves to “Resources” but the “Research” menu does not collapse. Note: the expanded menu will collapse if any Menu items are expanded later, however, collapsing should be automatic once the dropdown menu loses focus.

**Expected behavior**
Tabbing should work on menu and close after moving off.

See https://www.w3.org/WAI/ARIA/apg/patterns/menubar/ for proper behavior.

**Comment by trubach** [comment]
This is an issue with the Toolkit, it has been tested on https://builder3.toolkit.illinois.edu/components_toc/index.html and the same behavior happens. This was reported by Tim Offenstein. This issue should be brought over to the QC issues in the Toolkit project.

**Comment by trubach** [comment]
I've created an issue in the ilw-header-menu repo and it can be tracked at https://github.com/web-illinois/ilw-header-menu/issues/34
